roles/openshift_certificates/tasks/main.yml

```diff
@@ -0,0 +1,16 @@
+---
+- fail:
+    msg: "openshift_certificates_ca_host must be defined"
+  when: openshift_certificates_ca_host is not defined
+
+- fail:
+    msg: "openshift_certificates_action must be defined"
+  when: openshift_certificates_action is not defined
```

Comment:
I know I'm backtracking a bit here, but do we get enough code re-use out of consolidating master/node certs? I don't think I'm digging the use of openshift_certificates_action to drive the workflow. 

Maybe if the include behavior was triggered by membership in a list instead? `openshift_certificate_uses = ['master', 'node', 'etcd_server', 'etcd_client']`